universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -241,8 +241,9 @@ private CsvExportConfig generateDefaultCsvConfig(ImmutableList<AnswerData> answe
 
     // Add columns for each path to an answer.
     for (AnswerData answerData : answerDataList) {
-      if (answerData.questionDefinition().isEnumerator()) {
-        continue; // Do not include Enumerator answers in CSVs.
+      if (answerData.questionDefinition().isEnumerator()
+          || answerData.questionDefinition().getQuestionType() == QuestionType.FILEUPLOAD) {
```

Comment:
FileUploadQuestion originally returned an empty set for `allPaths()` and checked the existence of a specific path in `isAnswered`. As part of this change, I instead moved the path to be returned by `allPaths`.

ExporterService [calls `allPaths`](https://github.com/seattle-uat/civiform/blob/757d03f43b357d6c1e3a4b158104beebb70ced79/universal-application-tool-0.0.1/app/services/export/ExporterService.java#L360), which would now add a column for file uploads that I believe wasn't previously there. Because of this, I had actually thought that file uploads weren't included in CSV exports and thus excluded them.

Do you happen to know if there's an alternative way that file uploads are getting included?

If desired, I could also back out this change and restore the behavior in FileUploadQuestion of returning empty for `allPaths` and consulting a specific field for `isAnswered`.

Reply:
My thoughts here were misguided. It does appear that the file should be in the export. I still need to track down what's responsible for adding the content. I'll revert this and re-request review once I better understand the issue.

Reply:
File upload paths are created as part of the default CSV export and use a different method for determining the data to be shown. However, for demography export, the list of `allPaths` is consulted. For now, I've left the old FileUploadQuestion behavior and left a TODO for a follow-on CL since it likely could benefit from separate consideration in a separate PR if we further consolidate the behavior.